src/deploymentfrequency.ps1

```diff
@@ -1,17 +1,24 @@
-# TODO 
-# - There are tons of bugs
-# - no versioning/tags/releases - (just use @main tag for now)
-# - minimal error handling
-# - some rounding errors
-# - some questionable decisions made for the output. 
-
+#Parameters for the top level  deploymentfrequency.ps1 PowerShell script
 Param(
     [string] $ownerRepo,
     [string] $workflows,
     [string] $branch,
-    [Int32] $numberOfDays
+    [Int32] $numberOfDays,
+    [string] $ghPatToken = "",
+    [string] $ghActionsToken = ""#,
+    #[string] $ghAppToken 
 )
 
+#The main function
+function Main ([string] $ownerRepo,
+    [string] $workflows,
+    [string] $branch,
+    [Int32] $numberOfDays,
+    [string] $ghPatToken,
+    [string] $ghActionsToken#,
+    #[string] $ghAppToken 
+    ){
+
 #==========================================
```

Comment:
Need to add code indenting, but leaving it off until the last commit to make code reviewing easier.